protobuf/flow/access/access.proto

```diff
@@ -82,6 +82,12 @@ service AccessAPI {
 
   // GetNetworkParameters retrieves the Flow network details
   rpc GetNetworkParameters (GetNetworkParametersRequest) returns (GetNetworkParametersResponse);
+
```

Comment:
Created this corresponding issue - https://github.com/onflow/flow/issues/348 - I can take care of it.